plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -660,6 +711,54 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
         }
     }
 
+    private fun projectFieldAccess(fieldRef: FieldRef, childIdx: Int): NewCallBuilder{
+        val fieldAccessVars = mutableListOf<NewNodeBuilder>()
+        val leftOp = when(fieldRef){
+            is StaticFieldRef ->  fieldRef.fieldRef.declaringClass().type
+            is InstanceFieldRef -> fieldRef.base.type
+            else -> fieldRef.fieldRef.declaringClass().type
+        }
+
+        val fieldAccessBlock = NewCallBuilder()
+                .name(Operators.fieldAccess)
+                .code(fieldRef.toString()) // Review-TODO: Check Code is valid
+                .signature("(${leftOp.toQuotedString()}).(${fieldRef.toString()})") // Review-TODO: Signature of  [(Static/Instance) Field = Signature of Right Tree]
+                .methodFullName(Operators.fieldAccess)
+                .dispatchType(if(fieldRef is StaticFieldRef) DispatchTypes.STATIC_DISPATCH else DispatchTypes.DYNAMIC_DISPATCH)   // Review-TODO: How to handle which instance(Variable) is being used?
+                .dynamicTypeHintFullName(SootToPlumeUtil.createScalaList(leftOp.toQuotedString()))
+                .order(childIdx)
+                .argumentIndex(childIdx)
+                .typeFullName(leftOp.toQuotedString())
+                .lineNumber(Option.apply(currentLine))
+                .columnNumber(Option.apply(currentCol))
+                .apply{ fieldAccessVars.add(this) }
+        when(fieldRef){
+            is StaticFieldRef ->{ // Handle Static as Type_ref?
+                Pair(SootToPlumeUtil.createTypeRefVertex(fieldRef.type, currentLine, currentCol, 0),  // Type_Ref childIdx 1? cuz 0 is reserved.. for this as schema document
```

Comment:
You make a good point about the `childIdx` being 0 - this is a mistake on my part. You can make these `childIdx`s 1